src/System.Security.Cryptography.X509Certificates/tests/CertTests.cs

```diff
@@ -4,13 +4,21 @@
 
 using System.IO;
 using Xunit;
+using Xunit.Abstractions;
 
 namespace System.Security.Cryptography.X509Certificates.Tests
 {
-    public static class CertTests
+    public class CertTests
     {
+        private readonly ITestOutputHelper _log;
+
+        public CertTests(ITestOutputHelper output)
+        {
+            _log = output;
+        }
+
         [Fact]
-        public static void X509CertTest()
+        public void X509CertTest()
```

Comment:
I'm not sure about this, but I feel like it'll still run static methods when there's a mix of static and instance.  So you can keep the test class object instantiations down by leaving the rest static (xunit creates a new instance per instance-fact-method... possibly a new instance per instance-theory-case).